applications/SwimmingDEMApplication/python_scripts/swimming_DEM_solver.py

```diff
@@ -14,6 +14,114 @@ def Say(*args):
     Logger.Flush()
 
 class SwimmingDEMSolver(PythonSolver):
+
+    @classmethod
+    def GetDefaultParameters(cls):
+        # default settings string in json format
+        default_settings = Parameters("""{
+        "echo_level" : 1,
+
+        "gravity_parameters" : {
+            "modulus" : 9.81,
+            "direction" : [0.0, 0.0, -1.0]
+        },
+
+        "time_stepping" : {
+            "automatic_time_step" : true,
+            "time_step" : 0.001
+        },
+
+        "problem_data"  : {
+            "problem_name"  : "",
+            "echo_level" : 1,
+            "start_time" : 0.0,
+            "end_time"   : 1,
+            "parallel_type": "OpenMP",
+            "number_of_threads": 1
+        },
```

Comment:
The structure of the solver makes necessary to import the whole json, so for the moment we decided to validate all parameters of it. In the ProjectParameters.json there are not solver settings for dem only in the fluid_parameters section which are validate in the fluid solver.

Reply:
Yes, this is not a problem related to this PR, but to the structure of the SwimmingDEM solver, which is not just the coupling of two solvers, but of two analysis stages. This might need to be changed, but at the time when we started looking into the issue, it was not easy (problems with substepping etc.) and we went with a compromise solution. We will keep trying to converge to the Kratos ideal but, for the moment, the solver uses all this information, so I think this is not the place to change this.

Reply:
ok :)